core/lib/protobuf_config/src/proto/config/da_client.proto

```diff
@@ -8,9 +8,10 @@ message AvailConfig {
   optional string api_node_url = 1;
   optional string bridge_api_url = 2;
   optional uint32 app_id = 4;
-  optional uint64 timeout = 5;
-  optional uint64 max_retries = 6;
-  reserved 3; reserved "seed";
+  optional bool gas_relay_mode = 5;
```

Comment:
we can do oneof here so we don't need this flag and don't have any optionality

Reply:
Fixed, closing.